google/cloud/pubsub/schema_admin_connection.h

```diff
@@ -102,7 +102,8 @@ class SchemaAdminConnection {
  *     existing code that calls `MakeSchemaAdminConnection({})` from breaking,
  *     due to ambiguity.
  *
- * @deprecated Please use `MakeSchemaAdminConnection()` instead.
+ * @deprecated Please use the `MakeSchemaAdminConnection` function that accepts
+ *     `google::cloud::Options` instead.
```

Comment:
My thinking was that if someone wanted to call `MakeSchemaAdminConnection({})`, they should really be calling `MakeSchemaAdminConnection()`.

But if someone is looking through the documentation wondering which overload to use, it is probably best to redirect them to the `Options` one.